src/Microsoft.AspNet.SignalR.SqlServer/SqlSender.cs

```diff
@@ -43,14 +42,17 @@ public Task Send(IList<Message> messages)
                 return TaskAsyncHelper.Empty;
             }
 
-            var parameter = _dbProviderFactory.CreateParameter();
-            parameter.ParameterName = "Payload";
-            parameter.DbType = DbType.Binary;
-            parameter.Value = SqlPayload.ToBytes(messages);
+            using (var stream = new MemoryStream(SqlPayload.ToBytes(messages)))
```

Comment:
IIRC rather than wrapping `byte[]` with MemoryStream, you can simply set `Size = int.MaxValue` to get the same effect; the parameter should get sent as a `varbinary(max)`.